cloudbridge/cloud/providers/gce/resources.py

```diff
@@ -176,66 +177,80 @@ def zones(self):
 
 
 class GCEFirewallsDelegate(object):
+    NETWORK_URL_PREFIX = 'global/networks/'
```

Comment:
Are these two "constants" intended to be class private? If so, prefix them with an underscore.